test/plot-schema.json

```diff
@@ -13273,6 +13273,104 @@
      }
     },
     "role": "object",
+    "subtitle": {
+     "editType": "layoutstyle",
+     "font": {
+      "color": {
+       "editType": "layoutstyle",
+       "valType": "color"
+      },
```

Comment:
@emilykl Do any of the property values inherit from `layout.title`?
Also, `layout.title.subtitle.font.size` is 70% of `layout.title.font.size` unless set?